libs/event/src/lib/form/shell/shell.component.ts

```diff
@@ -91,33 +92,26 @@ export class EventFormShellComponent implements OnInit, OnDestroy {
 
   async save(options: { showSnackbar: boolean } = { showSnackbar: true }) {
     try {
-      // throw 'Parameter is not a number!';
-    if (this.form.valid && this.form.dirty) {
-      const value = this.form.value;
-      if (this.form.value.allDay) {
-        value.start.setHours(0, 0, 0);
-        value.end.setHours(23, 59, 59);
-      }
-      // value.start = undefined
-      await this.eventService.update(value);
-      this.form.markAsPristine();
-      this.cdr.markForCheck();
-      if (options.showSnackbar) {
-        this.snackBar.open('Event saved', 'CLOSE', { duration: 4000 });
+      if (this.form.valid && this.form.dirty) {
+        const value = this.form.value;
+        if (this.form.value.allDay) {
+          value.start.setHours(0, 0, 0);
+          value.end.setHours(23, 59, 59);
+        }
+        await this.eventService.update(value);
+        this.form.markAsPristine();
+        this.cdr.markForCheck();
+        if (options.showSnackbar) {
+          this.snackBar.open('Event saved', 'CLOSE', { duration: 4000 });
+        }
       }
+      return true;
     }
-    return true;
-  }
     catch (err) {
-      console.log(err)
-      this.snackBar.openFromTemplate(this.somethingWentWrongTemplate, { duration: 5000 });
+      this.snackBar.openFromComponent(SnackbarErrorComponent, { duration: 5000 });
```

Comment:
for specific message you can 
```suggestion
      this.snackBar.openFromComponent(SnackbarErrorComponent, { message: 'test message', duration: 5000 });
```